velox/docs/functions/spark/datetime.rst

```diff
@@ -70,6 +70,18 @@ These functions support TIMESTAMP and DATE input types.
 
     This is an alias for :func:`day_of_week`.
 
+.. function:: get_timestamp(string, dateFormat) -> timestamp
+
+    Returns timestamp by parsing ``string`` according to the specified ``dateFormat``.
+    The format follows Spark's
+    `Datetime patterns
+    <https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html>`_.
+    Returns NULL for parsing error or NULL input. ::
```

Comment:
When 'string' is invalid, this function return null. When dateFromat is invalid, it throws. Is this so? Would be nice to clarify and include examples of both cases.

Reply:
Yes, as confirmed spark code, invalid format specified by user will throw exception. Just documented this behavior and added such test cases.